cpp/ql/src/experimental/Security/CWE/CWE-570/WrongInDetectingAndHandlingMemoryAllocationErrors.qhelp

```diff
@@ -3,16 +3,19 @@
   "qhelp.dtd">
 <qhelp>
 <overview>
-<p>When using the <code>new</code> operator to allocate memory, you need to pay attention to the different ways of detecting errors. <code>::operator new(std::size_t)</code> throws an exception on error, whereas <code>::operator new(std::size_t, const std::nothrow_t &amp;)</code> returns zero on error. The programmer can get confused and check the error that occurs when allocating memory incorrectly. That can lead to an unhandled program termination or to a violation of the program logic.</p>
+<p>Different overloads of the <code>new</code> operator handle allocation failures in different ways:
```

Comment:
I'm not keen on the colon here, I think I'd prefer a simple full stop (or a colon followed by a list of bullet points perhaps?)

Reply:
Good point. I think I prefer the full stop (mostly just because it's simpler).

Reply:
Fixed in c67ab8f1f0dd170d3b6bbc13f39e2b39e009a346.